signer/src/testing/message.rs

```diff
@@ -107,9 +108,12 @@ impl fake::Dummy<fake::Faker> for super::WstsMessage {
     }
 }
 
-fn dummy_payload<P: Into<super::Payload> + fake::Dummy<fake::Faker>, R: rand::RngCore + ?Sized>(
+fn dummy_payload<
+    P: Into<message::Payload> + fake::Dummy<fake::Faker>,
+    R: rand::RngCore + ?Sized,
+>(
```

Comment:
style nit: in these kinds of situations, can we go with something like this style instead?
```rust
fn dummy_payload<P, R>(config: &fake::Faker, rng: &mut R) -> message::Payload
where
    P: Into<message::Payload> + fake::Dummy<fake::Faker>,
    R: rand::RngCore + ?Sized,
{
```
I find it much easier to read, but if you don't feel free to keep it as is.

Reply:
Definitely, I'll update.

Reply:
Done ✅